student-info-system/sis/models.py

```diff
@@ -466,7 +468,7 @@ class SectionStudent(models.Model):
     )
     grade = models.SmallIntegerField(
         choices=GRADES,
-        default=None,
+        default=GRADE_NG,
```

Comment:
not persuaded by this. I'd expect default to be None. Can you tell me more?

Reply:
fixed